apps/app/codegen.ts

```diff
@@ -3,7 +3,7 @@ import { CodegenConfig } from '@graphql-codegen/cli';
 const config: CodegenConfig = {
   //schema: 'http://clinical-api.tau.health:8080/graphql',
   schema: 'http://clinical-api.boson.health/graphql',
-  documents: ['apps/app/src/**/Settings/**/*.tsx'],
+  documents: ['apps/app/src/**/Settings/**/*.tsx', 'apps/app/src/views/components/TicketModal.tsx'],
```

Comment:
@marisbest2  and I talked about this, this might get messy adding single files but fine for now. Long term, we can add some sort of identifier to the graphql operation that identifies which schema it's pointing to.